custom/inddex/filters.py

```diff
@@ -0,0 +1,197 @@
+            ('urban', _('Urban')),
+            ('rural', _('Rural'))
+        ]
+
+
+class BreastFeedingFilter(BaseSingleOptionFilter):
+    slug = 'breastfeeding'
+    label = _('Breastfeeding')
+    default_text = _('All')
+
+    @property
+    def options(self):
+        # TODO: Check value in production apllication ?? refuse_de_rpondre
+        return [
+            ('yes', _('Yes')),
+            ('no', _('No')),
+            ('refuse_de_rpondre', _('Refuse to answer'))
+        ]
+
+
+class SupplementsFilter(BaseSingleOptionFilter):
+    slug = 'supplements'
+    label = _('Supplement Use')
+    default_text = _('All')
+
+    @property
+    def options(self):
+        return [
+            ('yes', _('Yes')),
+            ('no', _('No'))
+        ]
+
+
+class RecallStatusFilter(BaseSingleOptionFilter):
+    slug = 'recall_status'
+    label = _('Recall Status')
+    default_text = _('All')
+
+    @property
+    def options(self):
+        return [
+            ('Open', _('Not Completed')),
+            ('Completed', _('Completed'))
+        ]
+
+
+class GapDescriptionFilter(BaseSingleOptionFilter):
+    slug = 'gap_description'
+    label = _('Gap description')
+    default_text = _('All')
+
+    @property
+    def gaps_descriptions(self):
+        return [
+            ('conv factor available', 'conv factor available'),
+            ('fct data available', 'fct data available'),
+            ('no conversion factor available', 'no conversion factor available'),
+            ('no fct data available', 'no fct data available'),
+            ('not applicable', 'not applicable'),
+            ('using conversion factor from base term food code', 'using conversion factor from base term food code'),
+            ('using fct data from base term food code', 'using fct data from base term food code'),
+        ]
+
+    @property
+    def options(self):
+        return [
+            x for x in self.gaps_descriptions
+        ]
+
+
+class GapTypeFilter(BaseSingleOptionFilter):
+    slug = 'gap_type'
+    label = _('Gap type')
+    default_text = _('All')
+
+    @property
+    def options(self):
+        return [
+            ('fct', 'fct'),
+            ('conversion factor', 'conversion factor'),
+        ]
+
+
+class FoodCodeFilter(BaseSingleOptionFilter):
+    slug = 'food_code'
+    label = _('Food Code')
+    default_text = _('All')
+
+    @property
+    def food_codes(self):
+        return FoodCodeData(config={'domain': self.domain}).rows
+
+    @property
+    def options(self):
+        return [
+            (str(x), str(x)) for x in self.food_codes
+        ]
+
+
+class FoodBaseTermFilter(BaseSingleOptionFilter):
+    slug = 'food_base_term'
+    label = _('Food base term')
+    default_text = _('All')
+
+    @property
+    def food_base_terms(self):
+        return FoodBaseTermData(config={'domain': self.domain}).rows
+
+    @property
+    def options(self):
+        return [
+            (x, _(x)) for x in self.food_base_terms
```

Comment:
These terms won't be translated unless they're defined somewhere in the codebase.